ref_ts/static/ref_ts/rickshaw/README.md

```diff
@@ -1,207 +0,0 @@
-# Rickshaw
```

Comment:
@zhiyuli Also, no need to litter up the hydroshare repo (and static files) with readme documents for JS libraries.